testflinger_agent/client.py

```diff
@@ -55,6 +58,28 @@ def _requests_retry(self, retries=3):
         session.mount("https://", adapter)
         return session
 
+    def _configure_influx(self):
+        """Configure InfluxDB client using environment variables.
+
+        :return: InfluxDBClient object
+        """
+        # required influxdb env vars
+        req_env_vars = ["INFLUX_HOST", "INFLUX_USER", "INFLUX_PW"]
+
+        if not all(var in os.environ for var in req_env_vars):
+            return
+
+        host = os.environ.get("INFLUX_HOST")
+        port = int(os.environ.get("INFLUX_PORT", 8086))
+        user = os.environ.get("INFLUX_USER")
+        password = os.environ.get("INFLUX_PW")
+
+        influx_client = influxdb.InfluxDBClient(
+            host, port, user, password, self.influx_agent_db
+        )
+
```

Comment:
IMPORTANT: we need to ensure that the database is created here! if not, then we silently fail forever and it's not clear why!
ex. influx_client.create_database(self.influx_agent_db)